third_party/tango_api/tango_3d_reconstruction/include/tango_3d_reconstruction/tango_3d_reconstruction_api.h

```diff
@@ -47,10 +47,10 @@ typedef enum {
 /// Equivalent to those found in Android core/system/include/system/graphics.h.
 /// See TangoImageBuffer for a format description.
 typedef enum {
-  TANGO_3DR_HAL_PIXEL_FORMAT_RGBA_8888 = 1,       ///< RGBA 8888
-  TANGO_3DR_HAL_PIXEL_FORMAT_RGB_888 = 3,         ///< RGB 888
-  TANGO_3DR_HAL_PIXEL_FORMAT_YV12 = 0x32315659,   ///< YV12
-  TANGO_3DR_HAL_PIXEL_FORMAT_YCrCb_420_SP = 0x11  ///< NV21
+  TANGO_3DR_HAL_PIXEL_FORMAT_RGBA_8888 = 1,         ///< RGBA 8888
+  TANGO_3DR_HAL_PIXEL_FORMAT_RGB_888 = 3,           ///< RGB 888
+  TANGO_3DR_HAL_PIXEL_FORMAT_YCrCb_420_SP = 0x11,   ///< NV21
+  TANGO_3DR_HAL_PIXEL_FORMAT_DEPTH16 = 0x44363159,  ///< DEPTH16
```

Comment:
So these are changes from master? Would it make sense to have this header as a submodule from [tango examples c](https://github.com/googlesamples/tango-examples-c/blob/master/tango_3d_reconstruction/include/tango_3d_reconstruction_api.h) github?